package.json

```diff
@@ -41,6 +42,7 @@
     "electron": "./node_modules/.bin/electron"
   },
   "devDependencies": {
+    "@applitools/eyes-storybook": "^2.7.36",
```

Comment:
@tarynbillie please remove `^` from the version value (we use fixed dependencies versions) and make sure yarn.lock gets updated too.

Reply:
These changes have been made!